x-pack/plugins/cases/public/components/configure_cases/connectors_dropdown.test.tsx

```diff
@@ -36,192 +37,27 @@ describe('ConnectorsDropdown', () => {
-              />
-            </EuiFlexItem>
-            <EuiFlexItem
-              grow={false}
-            >
-              <span>
-                My Connector 2
-              </span>
-            </EuiFlexItem>
-          </EuiFlexGroup>,
-          "value": "resilient-2",
-        },
-        Object {
-          "data-test-subj": "dropdown-connector-jira-1",
-          "inputDisplay": <EuiFlexGroup
-            alignItems="center"
-            gutterSize="s"
-            responsive={false}
-          >
-            <EuiFlexItem
-              grow={false}
-            >
-              <Styled(EuiIcon)
-                size="m"
-                type="logoSecurity"
-              />
-            </EuiFlexItem>
-            <EuiFlexItem
-              grow={false}
-            >
-              <span>
-                Jira
-              </span>
-            </EuiFlexItem>
-          </EuiFlexGroup>,
-          "value": "jira-1",
-        },
-        Object {
-          "data-test-subj": "dropdown-connector-servicenow-sir",
-          "inputDisplay": <EuiFlexGroup
-            alignItems="center"
-            gutterSize="s"
-            responsive={false}
-          >
-            <EuiFlexItem
-              grow={false}
-            >
-              <Styled(EuiIcon)
-                size="m"
-                type="logoSecurity"
-              />
-            </EuiFlexItem>
-            <EuiFlexItem
-              grow={false}
-            >
-              <span>
-                My Connector SIR
-              </span>
-            </EuiFlexItem>
-          </EuiFlexGroup>,
-          "value": "servicenow-sir",
-        },
-        Object {
-          "data-test-subj": "dropdown-connector-servicenow-uses-table-api",
-          "inputDisplay": <EuiFlexGroup
-            alignItems="center"
-            gutterSize="s"
-            responsive={false}
-          >
-            <EuiFlexItem
-              grow={false}
-            >
-              <Styled(EuiIcon)
-                size="m"
-                type="logoSecurity"
-              />
-            </EuiFlexItem>
-            <EuiFlexItem
-              grow={false}
-            >
-              <span>
-                My Connector
-                 (deprecated)
-              </span>
-            </EuiFlexItem>
-            <EuiFlexItem
-              grow={false}
-            >
-              <Styled(EuiIconTip)
-                aria-label="This connector is deprecated. Update it, or create a new one."
-                color="warning"
-                content="This connector is deprecated. Update it, or create a new one."
-                size="m"
-                type="alert"
-              />
-            </EuiFlexItem>
-          </EuiFlexGroup>,
-          "value": "servicenow-uses-table-api",
-        },
-      ]
-    `);
+    expect(selectProps.options).toMatchSnapshot();
   });
 
-  test('it disables the dropdown', () => {
+  it('disables the dropdown', () => {
```

Comment:
I replaced all the redundant `test('it ` with the equivalent `it('` to make it easier to read.